recipe/meta.yaml

```
@@ -11,18 +11,21 @@ source:
 
 build:
   number: 1
-  skip: true  # [win]
+  #skip: true  # [win]
```

To  bump the build number, increase it here. You can remove the skip line completely.
```suggestion
  number: 2
```

---

Thanks. Yes, I am stuck on getting the build environment. Why Py2.7? I need >=VS2005 according to https://svn.apache.org/repos/asf/subversion/trunk/INSTALL, but could take anything later.

---

@zklaus BTW, what does the build # do? No hurry, it will be a few hours before resume

---

Python 2.7 is asked for in meta.yml in line 24. But Python 2.7 is end-of-lifed and hasn't been build in conda-forge for a long time. That's why the only available Python 2.7 packages are old packages that were build with old compilers. Now for ABI compatibility, conda-build asks for the old compiler, but doesn't find it anymore because that, too, is gone.

But reading the document you linked, it seems that Python 3 should work, so I would suggest to remove the version constraint on Python and switch back to the normal compiler selection mechanism.

---

> @zklaus BTW, what does the build # do? No hurry, it will be a few hours before resume

The build number is part of the version of the conda-forge package. Basically that consists of the upstream version, in this case 1.14.2 that was decided by the subversion developers, and something that tells us about the packaging. The build number is part of that, so always when we change the packaging, but not the underlying upstream project, we increase the build number so that we know which build is newer. The most common reason to create a new build is when a dependency was updated and the present project needs to be recompiled and relinked against the new version of the dependency.